File: youqugou-web-shop/src/main/webapp/js/service/homeService.js
/**
 * Author:Mr Liu
 * Date:2018-12-28
 */
app.service("homeService", function ($http) {
    this.searchTime = function () {
        return $http.get("../home/searchTime.do");
    }
    this.getSales = function (year, month, timeBucket) {
        return $http.get("../home/getSales.do?year=" + year + "&month=" + month + "&timeBucket=" +timeBucket);
    }
});

File: youqugou-interface/src/main/java/cn/itcast/core/service/HomeService.java
package cn.itcast.core.service;

import java.util.Date;
import java.util.List;

/**
 * Author:Mr Liu
 * Date:2018-12-28
 */
public interface HomeService {
    Date searchTime(String sellerId);

    List<Integer> getSales(List<Date> list, String sellerId);
}
